src/App.scss

```diff
@@ -0,0 +1,14 @@
+$purple: #2A2283;
+$purple-variant: #2f155c;
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    transition: .5s;
```

Comment:
I'd recommend to add transitions only to elements that use animations

Reply:
Attended